src/bitgo.js

```diff
@@ -1089,7 +1094,12 @@ BitGo.prototype.authenticate = function(params, callback) {
     return this.reject('already logged in', callback);
   }
 
-  const request = this.post(this.url('/user/login'));
+  let request;
+  if (params.microservices && this._microservicesUrl) {
```

Comment:
Why does another param need to be passed to auth to microservices here? Can't we know we need to auth to microservices based on whether or not this_.microservicesUrl is set?

Reply:
I was thinking it as 'maybe we want to use the microservicesUrl for other calls but not necessarily for authenticate, so we add another guarantee that we actually want it. however this is a side case and I'm fine with just removing it.

Reply:
Yeah, I don't think we need this. You can always `delete bitgo._microservicesUrl` if you really want to start authing the normal way after the object is created for some reason.